### DISCUSSION TITLE
Base Period in Event Study Plots

### DISCUSSION THREAD

**Discussion by bcallaway11** [bcallaway11/did#112] [Q&A] [answered]
It would be nice to have the option of making all event time effects relative to base period in the immediate period before actually participating in the treatment.  The way we do it now probably makes more sense, but this would be a feature to make results directly comparable to "event study plots" coming from regressions.

**Comment by nateapathy** [comment]
> The way we do it now

Thanks for this awesome set of tools! Question about how the event study plots are currently done as you reference here. Is the current relative period min(t)-1? e.g. for the examples in the [CRAN vignettes](https://cran.r-project.org/web/packages/did/vignettes/did-basics.html), is t=-3 the relative period?

And just to confirm across explainers, in the [pre-testing vignette](https://bcallaway11.github.io/did/articles/pre-testing.html), it's difficult to tell if the estimates plotted in the canonical ES plot at t=-2 and t=-3 are different from the t=-1 and t=-2 estimates in ggdid() with dynamic effects. So there are 6 points in the ES plot but only 5 in the ggdid() plot, am I correct in understanding that t=-3 is also serving as the reference period in the ggdid() plot and being left out rather than explicitly plotted as t=-1 is in the ES plot?

Thanks again! Would definitely make use of this feature if you're considering adding it :-)

**Comment by bcallaway11** [comment]
Hi Nate,

For us, the base period is usually (g-1), meaning the period right before group g becomes treated. 

The exception to this is in pre-treatment periods.  In that case, the base period is just the period right before the current one.

This is also the difference between the event study regression and our event study.  The event study regression (typically) uses the base period of (g-1) for all periods.  That’s why there is the extra estimate at e=-3 in the pre-testing vignette in that case.  

Brant

**Comment by bcallaway11** [comment, marked as answer]
As of versions `2.1.1`, we have support both for "varying" base periods (which is the default and was the only choice for a long time --- and, for what it's worth, it's the one I tend to prefer) and "universal" base periods (as in event study regressions).  A longer discussion is available in this [post](https://bcallaway11.github.io/posts/event-study-universal-v-varying-base-period).
